client/my-sites/themes/theme-preview.jsx

```diff
@@ -95,7 +99,13 @@ class ThemePreview extends Component {
 		if ( ! secondaryButton ) {
 			return;
 		}
-		const buttonHref = secondaryButton.getUrl ? secondaryButton.getUrl( this.props.themeId ) : null;
+
+		const styleVariationOption = this.getStyleVariationOption();
+		let buttonHref = secondaryButton.getUrl ? secondaryButton.getUrl( this.props.themeId ) : null;
+		if ( buttonHref && styleVariationOption ) {
+			buttonHref += `&style=${ styleVariationOption.slug }`;
```

Comment:
I'd prefer to use `style_variation` to align the core behavior. For example, the core uses `style_variation` to preview the styles. See https://wp-themes.com/twentytwentythree/?style_variation=aubergine

Reply:
Besides, I don't think we can assume the URL ends with the query string.